<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTimer</class>
 <widget class="QDialog" name="MainTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pomodoro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/pomodoro.png</normaloff>:/icons/pomodoro.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;
color:black;
padding-left:30px;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_date_time">
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-left:20px;
</string>
     </property>
     <property name="text">
      <string>Time Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="info_label">
     <property name="styleSheet">
      <string notr="true">color:rgba(153, 153, 153, 1);
padding:10px 60px;
margin-left:10px;
</string>
     </property>
     <property name="text">
      <string>5 min Break will start in every 25 min</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pauseResume">
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
color:white;
background-color:black;
margin: 0px 125px;

padding:12px 10px;</string>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
